tests/fips/mozilla_nss/nss_smoke.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's NSSFips tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Smoke test for NSS on FIPS sytems
+# Maintainer: qa-c team <qa-c@suse.de>
+#             QE Security
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use version_utils qw(is_sle is_transactional);
+use transactional qw(trup_call process_reboot);
+use serial_terminal 'select_serial_terminal';
+use utils 'zypper_call';
+
+sub run {
+    select_serial_terminal;
+    my $zypper_options;
+    my $pass_file = '/root/password.txt';
+    my $seed_file = '/root/seedfile.dat';
+    my $cert_file = '/root/cert9.cer';
+    my $nssdb_dir = '/root/nssdb';
+    my $cmd = "\"CN=Daniel Duesentrieb3,O=Example Corp,L=Mountain View,ST=California,C=DE\" -d \"$nssdb_dir\" -o $cert_file -f $pass_file -z $seed_file";
+
+    if (get_var('FIPS_CERT_15SP4')) {
+        zypper_call('ar ' . get_required_var('MOZILLA_NSS_DEVEL_REPO'));
+        zypper_call("--gpg-auto-import-keys ref", 180);
+        record_info('repos', script_output('zypper lr -u'));
+        $zypper_options = "--from 1";
+    }
+
+    if (is_transactional) {
+        trup_call("pkg install $zypper_options mozilla-nss mozilla-nss-tools");
+        process_reboot(trigger => 1);
+    } else {
+        zypper_call("in $zypper_options mozilla-nss mozilla-nss-tools");
+    }
+    record_info('mozilla-nss', script_output('rpm -q mozilla-nss'));
+    record_info('mozilla-nss', script_output('rpm -q mozilla-nss-tools'));
+
+    assert_script_run('cat /dev/urandom | head -n 120 > /root/seedfile.dat');
+    record_info('seedfile', script_output('cat /root/seedfile.dat'));
+    assert_script_run('echo "" > ' . $pass_file);
```

Comment:
is this about creating an empty file? or the quotes are required?
If it is about to create an empty file, please use `touch`